BASE/src/Microsoft.ApplicationInsights/Channel/Transmission.cs

```diff
@@ -133,6 +134,47 @@ public ICollection<ITelemetry> TelemetryItems
             get; private set;
         }
 
+        /// <summary>
+        /// Gets a value indicating whether flush is async.
+        /// </summary>
+        public bool HasFlushTask => this.flushTaskCompletionSource != null && !this.flushTaskCompletionSource.Task.IsCompleted;
```

Comment:
Would this syntax work here: 

`!this.flushTaskCompletionSource?.Task.IsCompleted;`

?

Reply:
It gives syntax error as we have .NET45 build